### PR TITLE
[FAQ] How to Remote Debug a CF App

### PR DESCRIPTION
**Pull request by Johannes-Schneider**
## What Has Changed?

This PR adds a FAQ entry about how to enable remote debugging on CF.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/java/faq.mdx [hunk 1]
```diff
@@ -58,6 +58,17 @@ You can also create an issue on the SAP Cloud SDK external GitHub repository or
 
 Choose `XX-S4C-SDK` if you are reporting an issue via Service Now.
 
+### How Do I Debug My Cloud Foundry Application?
```
Comment (artemkovalyov) [inline docs/java/faq.mdx:61]:
```suggestion
### How Do I Remotely Debug My App on BTP Cloud Foundry?
```

**Review by artemkovalyov — APPROVED** [review]
LGTM. Check my suggestion for the header.

File: docs/java/faq.mdx [hunk 2]
```diff
@@ -58,6 +58,17 @@ You can also create an issue on the SAP Cloud SDK external GitHub repository or
...
+
+Remote debugging your SAP BTP Cloud Foundry application can be achieved by following this [Blog Post's](https://blogs.sap.com/2019/07/24/remote-debugging-on-cloud-foundry/) instructions.
```
Comment (artemkovalyov) [inline docs/java/faq.mdx:63]:
```suggestion
Use these CF CLI instructions to activate debugging for your App.

**TomEE (when using java buildpack)**
```bash
cf ssh application -c "export JAVA_PID=`ps -C java -o pid=` && app/META-INF/.sap_java_buildpack/sap_machine_jdk/bin/jcmd $JAVA_PID VM.start_java_debugging"
cf ssh application -N -T -L 8000:127.0.0.1:8000
```
**Spring**
```bash
cf set-env application JBP_CONFIG_DEBUG "{enabled: true}"
cf restage application
cf ssh -N -T -L 8000:localhost:8000 application
```
Check out this [blog post](https://blogs.sap.com/2019/07/24/remote-debugging-on-cloud-foundry/) for step-by-step remote debugging instructions. You'll also learn how to get debug information into IntelliJ IDEA. 
```

**Review by artemkovalyov — COMMENTED** [review]
Another suggestion after reply by @newtork

**Comment by Johannes-Schneider** [comment]
Thanks for the suggestions, @artemkovalyov. I incorporated both of them.

**Pull request merged**